### PR TITLE
[Kohei] Assignment 2

### PR DESCRIPTION
**Pull request by axross**

### CODE REVIEW

**Comment by axross** [comment]
@itzderr Can you review the code? Thanks!

File: src/main/java/ca/ciccc/Assignment2.java [hunk 1]
```diff
@@ -90,9 +152,18 @@ public boolean tripleIncreasingOrder(int[] nums) {
...
+        for (int i = 0; i < nums.length; ++i) {
+            if (nums[i] % 2 == 0) {
+                evens += 1;
+            } else {
+                odds += 1;
+            }
+        }
 
-        return false;
+        return evens == odds;
```
Comment (Enrique92) [inline src/main/java/ca/ciccc/Assignment2.java]: You have to return true or false depending if is even or odds.

File: src/main/java/ca/ciccc/Assignment2.java [hunk 2]
```diff
@@ -33,9 +51,20 @@ public String numberLoops(int n) {
...
+            if (nums[i] == 1) {
+                numberOfOnes += 1;
+            }
+
+            if (nums[i] == 7) {
+                numberOfSevens += 1;
+            }
+        }
+
+        return numberOfOnes > numberOfSevens;
```
Comment (Enrique92) [inline src/main/java/ca/ciccc/Assignment2.java]: This one is the same return true if the number of 1s is greater than the number of 7s

File: src/main/java/ca/ciccc/Assignment2.java (hunk 1 above)
Comment (axross) [reply src/main/java/ca/ciccc/Assignment2.java]: @Enrique92 Thanks for a comment! I think `evens == odds` will be a boolean value. So it's fine with only returning it.

File: src/main/java/ca/ciccc/Assignment2.java (hunk 2 above)
Comment (axross) [reply src/main/java/ca/ciccc/Assignment2.java]: Thanks. However, this is the same as https://github.com/itzderr/Assignment2-Java/pull/2#discussion_r260456811.

Comment (amagin006) [reply src/main/java/ca/ciccc/Assignment2.java]: yes, it seems to be ok.

**Pull request closed without merging**